.circleci/config.yml

```diff
@@ -556,18 +558,23 @@ jobs:
               wget \
               xdg-utils
       - run:
-          name: Run tests on libs
-          command: yarn test-libs -w 3 --shard "$(($CIRCLE_NODE_INDEX + 1))/$CIRCLE_NODE_TOTAL"
+          name: Run tests on libs-components
+          command: yarn workspace lib-components run test -w 3 --shard "$(($CIRCLE_NODE_INDEX + 1))/$CIRCLE_NODE_TOTAL"
+      - run:
+          name: Run tests on lib-classroom
+          command: yarn workspace lib-classroom run test -w 3 --shard "$(($CIRCLE_NODE_INDEX + 1))/$CIRCLE_NODE_TOTAL"
       - run:
           name: Run tests on lti site
-          command: yarn test-lti -w 3 --shard "$(($CIRCLE_NODE_INDEX + 1))/$CIRCLE_NODE_TOTAL"
+          command: yarn workspace marsha run test -w 3 --shard "$(($CIRCLE_NODE_INDEX + 1))/$CIRCLE_NODE_TOTAL"
       - run:
           name: Run test on standalone site
-          command: yarn test-site
+          command: yarn workspace standalone_site run test
```

Comment:
Why not sharding these tests ?

Reply:
CRA uses Jest in a different way, the shard option doesn't work.

```
cross-env CI=true react-scripts test -w 3 --shard 1/4
● Unrecognized CLI Parameter: Unrecognized option "shard".
```

Reply:
I think they use a too older version without the shard option